### ISSUE TITLE
JourneyMap on a Server Fail

### ISSUE THREAD

**Issue by MeinKraft** [TeamJM/journeymap#18]
#### Instructions for Submitting an Issue
1. Ask for help on Discord (https://discord.gg/eP8gE69) #support if you're unsure this is a new issue.
2. Please search for existing issues before submitting a new one.
3. Please add the correct MC<version> label to your issue. Example: "[1.12.2] Waypoints are upside down"
4. Completely fill out the template below. Failure to do so is likely to get the issue closed immediately.
5. Prove you read these instructions by deleting this section before submitting. ;)
---
  
#### JourneyMap Version:

 journeymap-1.12.2-5.6.0b2

#### Forge Version:

1.12.1-14.23.1.2589

#### Describe the problem in repeatable steps:

1. copy mod to server and restart server
2. server shuts down with error report below
3. also I think your display time is calculated wrong, Project Ozone Lite displays different time to your map

#### Relevant errors from `.minecraft/journeymap/journeymap.log`:

https://pastebin.com/xjmbGnqJ

**Comment by mysticdrew** [comment]
use forge 14.23.1.2555 that is the recommended, you should never be using the latest, latest forge is essentially an alpha.

**Comment by PixelmonPRO** [comment]
@mysticdrew forge 14.23.1.2555: https://gist.github.com/PixelmonPRO/b0190315ba79af6cfba8ff8d0036cf2f

**Comment by SPK44** [comment]
I ran into this issue too. Rolling back to journeymap-1.12.2-5.5.2, with the same server and configurations produced no error. I also experienced this issue only after the second time of running the server.

**Comment by Boymann** [comment]
[crash-2018-01-16_21.56.54-server.txt](https://github.com/TeamJM/journeymap/files/1636783/crash-2018-01-16_21.56.54-server.txt)
Here is my crash report too. And as previously said, previous journeymap worked on any forge version. This version needs its configs deleted on each server start to not crash the server.

**Comment by smmmadden** [comment]
so to go from 1.12.2-5.5.2 to 1.12.2-5.6.0b2 deleting the server & client /journeymap folder allowed the server to startup and not crash.  At least this worked for me using the below setup. :-)
    Minecraft: 1.12.2
    SpongeAPI: 7.1.0-SNAPSHOT-dfc5c4a
    SpongeForge: 1.12.2-2555-7.1.0-BETA-2842
    Minecraft Forge: 14.23.1.2599

**Comment by seeker2921** [comment]
I am experiencing this as well, however, I am presently having an issue after downgrading to 5.5.2, now the server starts, but clients get no main map, it launches when hitting "J" but no icons, map, or the like.

This is on 5.5.2 and forge 2555.

There are no error messages in logs. it may be a mod conflict and I will attempt to isolate that and report back.

**Comment by smmmadden** [comment]
yeah, unfortunately it seems after every restart I have to delete the journeymap folder and remove the entry for it our of the Mercurius.cfg and then everything seems happy.

**Comment by Dunkh4n** [comment]
JourneyMap Version: journeymap-1.12.2-5.6.0b2
Forge Version: 14.23.1.2587
Describe the problem in repeatable steps:

I play on ATM 3 (All The Mods), whit the last journeyMap version (same problem with the default version in the modpack, can't remember it tough)
My server always crash when i restart it.
I have to delete all the file in journeymap\server\5.6, and after that it start ok.

**Comment by mysticdrew** [comment]
Closing all 5.6.0 issues. Please downgrade to 5.5.2 as 5.6.0 is no longer publicly available.
